src/layouts/index.jsx

```diff
@@ -181,26 +182,38 @@ class Layout extends React.Component {
     const investigation = contextInvestigation || envInvestigation;
 
     return (
-      <>
-        <SEO title={investigationTitle || investigation || 'Investigation'} />
-        <Header
-          logo={investigation !== 'ngss-solar-system' ? logo : null}
-          pageNumber={this.getCurrentPageNumber(pages, pageId)}
-          tocVisability={tocIsOpen}
-          toggleToc={investigation && this.toggleToc}
-          {...{ sections, investigationTitle, totalPages }}
-        />
-        {investigation && (
-          <TableOfContents
-            visible={tocIsOpen}
-            toggleToc={this.toggleToc}
-            investigation={investigation}
-            isAll={!envInvestigation || envInvestigation === 'all'}
-            pages={pages}
-          />
+      <Translation>
```

Comment:
to get the investigation title translated this root layout was wrapped in the `Translation` component to give it access to the `t` method. The normal HOC pattern could not be applied here because this page is already wrapped in the `StaticQuery` HOC which doesn't use normal children but uses a `render` method that won't support the `withTranslation` HOC